sections/06_discussion.md

```diff
@@ -210,6 +210,184 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation? DeepChem and DragoNN are worth
 discussing here.*
 
-### Transfer learning/transferability of features
+### Multimodal, multi-task, and transfer learning
```

Comment:
This is how I saw these terms used most of the times in literature. Wikipedia also uses "Multimodal" and "Multi-task". So I'd say yes.